libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -267,6 +273,18 @@ private String registerSFDCPushNotification(String registrationId, UserAccount a
             fields.put(CONNECTION_TOKEN, registrationId);
             fields.put(SERVICE_TYPE, ANDROID_GCM);
             fields.put(APPLICATION_BUNDLE, SalesforceSDKManager.getInstance().getAppContext().getPackageName());
+
+            // Adds community ID to the registration payload to allow scoping of notifications per community.
+            final String communityId = UserAccountManager.getInstance().getCurrentUser().getCommunityId();
```

Comment:
This piece is for scoping of notifications per community, which is now supported for encrypted push notifications.